epytope/EpitopePrediction/External.py

```diff
@@ -463,22 +461,26 @@ def parse_external_result(self, file):
         :return: A dictionary containing the prediction results
         :rtype: dict
         """
-        result = defaultdict(defaultdict)
+        scores = defaultdict(defaultdict)
+        ranks = defaultdict(defaultdict)
         f = csv.reader(open(file, "r"), delimiter='\t')
-        pos_factor = 3
       # alleles = map(lambda x: x.split()[0], filter(lambda x: x.strip() != "", f.next()))
       # f.next()
-        alleles = [x.split()[0] for x in [x for x in next(f) if x.strip() != ""]]
+        alleles = [Allele(x.split()[0][:5]+'*'+x.split()[0][5:7]+ ':' +x.split()[0][7:]) for x in [x for x in next(f) if x.strip() != ""]]
```

Comment:
The old solution did not work for me. I think the output was `A0201` and converting it to an Allele Obj did not work since it needs `*, :` in there

Reply:
Thinking about this again, I don't really get why the allele output of `NetMHC 4.0` should have changed. That doesn't make sense to me.

Reply:
Since every netMHC tool seems to have a different HLA nomenclature I decided to transform the allele strings into `Allele Obj` in the parser as discussed.

Reply:
Now I got and now I understand why it does not work anymore using the old expression. I think it would be nice to introduce a function for the "External prediction classes" like `to_allele` which handles the conversation, but that doesn't have to be part of this PR.

Reply:
Good suggestion! Noted in the backlog